code/controllers/subsystem/persistence.dm

```diff
@@ -410,3 +410,31 @@ SUBSYSTEM_DEF(persistence)
 		if(!ending_human.client)
 			return
 		ending_human.client.prefs.save_character()
+
+// Removes paintings with invalid md5
+/datum/controller/subsystem/persistence/proc/prune_persistent_paintings()
+	var/list/invalid = list()
+	for(var/pid in SSpersistence.paintings)
+		var/list/pruned_category = list()
+		var/list/current = SSpersistence.paintings[pid]
+		if(!current)
+			current = list()
```

Comment:
why not just return here?

Reply:
Because more stuff happens below

Reply:
It's just to ensure even empty category keys end up in final json.

Reply:
Oh I see. Thanks